src/main/java/com/yunhalee/walkerholic/activity/controller/ActivityController.java

```diff
@@ -1,46 +1,59 @@
 package com.yunhalee.walkerholic.activity.controller;
 
-import com.yunhalee.walkerholic.activity.dto.ActivityCreateDTO;
-import com.yunhalee.walkerholic.activity.dto.ActivityDTO;
+import com.yunhalee.walkerholic.activity.dto.ActivityRequest;
+import com.yunhalee.walkerholic.activity.dto.ActivityResponse;
+import com.yunhalee.walkerholic.activity.dto.ActivityDetailResponse;
 import com.yunhalee.walkerholic.activity.service.ActivityService;
+import java.io.IOException;
 import lombok.RequiredArgsConstructor;
 import org.springframework.web.bind.annotation.*;
 import org.springframework.web.multipart.MultipartFile;
 
 import java.util.List;
 
 @RestController
-@RequiredArgsConstructor
 public class ActivityController {
 
     private final ActivityService activityService;
 
-  @PostMapping("/activity/save")
-    public ActivityCreateDTO saveActivity(@RequestParam(value = "id", required = false) Integer id,
-        @RequestParam("name") String name,
+    public ActivityController(ActivityService activityService) {
+        this.activityService = activityService;
+    }
+
+    @PostMapping("/activities")
+    public ActivityResponse create(@RequestParam("name") String name,
         @RequestParam("score") Integer score,
         @RequestParam("description") String description,
-        @RequestParam(value = "multipartFile", required = false) MultipartFile multipartFile) {
+        @RequestParam(value = "multipartFile", required = false) MultipartFile multipartFile)
+        throws IOException {
+        ActivityRequest activityRequest = new ActivityRequest(name, score, description);
+        return activityService.create(activityRequest, multipartFile);
+    }
 
-        ActivityCreateDTO activityCreateDTO = new ActivityCreateDTO(id, name, score, description);
-        return activityService.saveActivity(activityCreateDTO, multipartFile);
+    @PutMapping("/activities/{id}")
+    public ActivityResponse update(@PathVariable("id") Integer id,
+        @RequestParam("name") String name,
+        @RequestParam("score") Integer score,
+        @RequestParam("description") String description,
```

Comment:
```suggestion
        @RequestBody ActivityRequest activityRequest
```
description과 같은 데이터를 굳이 request parameter로 받는 이유가 있으실까요?

Reply:
해당 부분 확인하여 RequestBody로 변경하였습니다!